lib/Common/Memory/arm64/XDataAllocator.cpp

```diff
@@ -98,14 +163,19 @@ void XDataAllocator::Register(XDataAllocation * xdataInfo, intptr_t functionStar
 void XDataAllocator::Unregister(XDataAllocation * xdataInfo)
 {
 #ifdef _WIN32
-    NtdllLibrary::Instance->DeleteGrowableFunctionTable(xdataInfo->functionTable);
+    // Delete the table
+    if (AutoSystemInfo::Data.IsWin8OrLater())
+    {
+        NtdllLibrary::Instance->DeleteGrowableFunctionTable(xdataInfo->functionTable);
+    }
+    else
+    {
+        BOOLEAN success = RtlDeleteFunctionTable(&xdataInfo->pdata);
+        Assert(success);
+    }
```

Comment:
nit: I think ARM64 is only supported on Win10 and above so this else can be removed.